packages/ethereum-erc20-provider/lib/EthereumErc20Provider.js

```diff
@@ -4,8 +4,8 @@ import { BigNumber } from 'bignumber.js'
 import Provider from '@liquality/provider'
 import { padHexStart } from '@liquality/crypto'
 import {
-  ensureAddressStandardFormat,
-  ensureHexEthFormat
+  toLowerCaseWithout0x,
```

Comment:
Yea this is kind of what I meant when i said the naming might start to mean more than just the 0x 😄 
I hope we don't start to have sentences describing what a function does eventually... This is why calling it just `standard` or `normalised` is a better naming in my opinion.